src/com/magento/idea/magento2plugin/actions/generation/data/DeleteEntityControllerFileData.java

```diff
@@ -26,12 +27,14 @@ public DeleteEntityControllerFileData(
             final @NotNull String entityName,
             final @NotNull String moduleName,
             final @NotNull String acl,
-            final @NotNull String entityId
+            final @NotNull String entityId,
+            final boolean isDeleteCommandHasInterface
```

Comment:
Let's replace this one to a more readable version: `hasDeleteCommandInterface` or `deleteCommandHasInterface`.

```suggestion
            final boolean hasDeleteCommandInterface
```

Reply:
Done.